### ISSUE TITLE
Save likes and reposts to soundcloud

### ISSUE THREAD

**Issue by rocketinventor** [ptgamr/upnext#23]
Hi,
I would really appreciate it if I could like a track on Soundcloud from inside of the extension and, therefore support the artist that I listen to.

**Comment by ptgamr** [comment]
Hmn... I made this extension with simplicity in mind. I don't have plan for deep integration with SoundCloud. You can always go to the song by follow the link provided by the extension and give the support for artists. 

![](http://i.imgur.com/1MZpDMx.png)

**Comment by rocketinventor** [comment]
No, no no, it wouldn't take away from the simplicity of the extension.
All that would be required is a one time authentication with soundcloud,
and then the likes would just automatically saved to soundcloud.
Also, this could have the added benefit of automatically importing the
users playlists.
The main reason I downloaded this extension is so that I wouldn't have to
keep soundcloud open in a tab.
However, if there is no integration and I have to open a new tab every time
I wanted to like a song then, the point is completely defeated an I see no
point of keeping installed :(
Also, what is the point of the likes function if they are not useable
elsewhere?

On Sat, Jul 4, 2015 at 12:02 AM Anh Trinh <notifications@github.com> wrote:

> Closed #23 <https://github.com/ptgamr/upnext/issues/23>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ptgamr/upnext/issues/23#event-347701255>.
>
-- 
Elliot Gerchak,
Owner of dementedLab.com

**Comment by ptgamr** [comment]
Hmn... I guess its reasonable. I will give it a second thought

**Comment by rocketinventor** [comment]
Great, thanks for considering!

On Sun, Jul 5, 2015 at 7:24 PM Anh Trinh <notifications@github.com> wrote:

> Hmn... I guess its reasonable. I will give it a second thought
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ptgamr/upnext/issues/23#issuecomment-118676268>.
>
-- 
Elliot Gerchak,
Owner of dementedLab.com
